package.json

```diff
@@ -145,7 +145,8 @@
     "micromatch": "^4.0.7",
     "axios": "^1.7.4",
     "fast-xml-parser": "^4.4.1",
-    "ws": "^8.17.1"
+    "ws": "^8.17.1",
+    "tar-fs": "2.1.3"
```

Comment:
Should this be `^2.1.3` or should it be pinned only to that specific patch version?